frontend/src/entry-forms/Transaction.svelte

```diff
@@ -41,34 +42,14 @@
     }
   }
 
-  /// Extract tags and links that can be provided in the narration <input>.
-  function onNarrationChange({
-    currentTarget,
-  }: {
-    currentTarget: HTMLInputElement;
-  }) {
-    const { value } = currentTarget;
-    entry.tags = [...value.matchAll(TAGS_RE)].map((a) => a[1] ?? "");
-    entry.links = [...value.matchAll(LINKS_RE)].map((a) => a[1] ?? "");
-    entry.narration = value
-      .replaceAll(TAGS_RE, "")
-      .replaceAll(LINKS_RE, "")
-      .trim();
+  /// Extract tags and links that can be provided in the narration.
+  function onNarrationChange() {
+    entry.tags = [...entry.narration.matchAll(TAGS_RE)].map((a) => a[1] ?? "");
+    entry.links = [...entry.narration.matchAll(LINKS_RE)].map(
+      (a) => a[1] ?? "",
+    );
   }
 
-  /// Output tags and links in the narration <input>
-  function combineNarrationTagsLinks(e: Transaction): string {
```

Comment:
this needs to be re-added. we need to add the tags and links here

Reply:
It's also present again, but it is only executed once a selection is made.